modules/layers/src/text-layer/text-layer.js

```diff
@@ -188,14 +188,27 @@ export default class TextLayer extends CompositeLayer {
       const text = getText(object, objectInfo);
       if (text) {
         const letters = Array.from(text);
-        const offsets = [0];
+
         let offsetLeft = 0;
+        let offsetTop = 0;
+        let lineHeight = 0;
+
+        // offsetX, offsetY
+        const offsets = [[0, 0]];
```

Comment:
My understanding is that there is one offset value for each character. Why create a shared array instead of just assigning the value to each datum?